plugins/hls-hlint-plugin/hls-hlint-plugin.cabal

```diff
@@ -2,11 +2,11 @@ cabal-version: 2.2
 name:          hls-hlint-plugin
 version:       0.1.0.0
 synopsis:      Hlint integration plugin with Haskell Language Server
-description:   Please see README.md
+description:   Please see Haskell Language Server Readme (https://github.com/haskell/haskell-language-server#readme)
 license:       Apache-2.0
 license-file:  LICENSE
 author:        Many,TBD when we release
```

Comment:
I guess we should to change this one for sure 😄 
If we got a collective description, we could use it here